generic_expression_patterns_modules/download_recount2_data.R

```diff
@@ -1,82 +1,77 @@
 ## Run this once to setup environment
 ## Used R 3.6.3
-#if (!requireNamespace("BiocManager", quietly = TRUE))
-#  install.packages("BiocManager")
 
-#BiocManager::install("recount")
```

Comment:
I deleted this line because `recount` has been installed in conda env.

Reply:
I thought I tried tried to remove this and it didn't work. But if you have tried removing it and its not needed that's great. It was a while ago so I might be misrembering

Reply:
After creating a brand new conda env, I tested `library("recount")` command in a R session and got the following error:
```
Error: package or namespace load failed for ‘recount’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/ubuntu/miniconda3/envs/dhu_test/lib/R/library/stringi/libs/stringi.so':
  libicui18n.so.64: cannot open shared object file: No such file or directory
```
More details on this error can be found here:
https://github.com/conda-forge/r-stringi-feedstock/issues/20
To make a long story short, we need to explicitly install `r-stringi`  in `r` channel in `environment.yml`. I already tested it in another conda env. I will add the line in `environment.yml`.